core/src/apps/management/backup_device.py

```diff
@@ -5,8 +5,8 @@
 from apps.common import mnemonic, storage
 from apps.management.reset_device import (
     backup_bip39_wallet,
-    backup_group_slip39_wallet,
-    backup_slip39_wallet,
+    backup_slip39_advanced_wallet,
+    backup_slip39_basic_wallet,
```

Comment:
Maybe let's strip the `wallet`? It seems to like there is basic and advanced wallet :). So just  `backup_slip39_advanced` and `backup_slip39_basic`?

Reply:
done